src/test/java/giis/eshopcontainers/e2e/functional/LoggedUserTest.java

```diff
@@ -0,0 +1,31 @@
+package giis.eshopcontainers.e2e.functional;
+
+import giis.eshopcontainers.e2e.functional.common.BaseLoggedTest;
+import giis.eshopcontainers.e2e.functional.common.exceptions.ElementNotFoundException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+class LoggedUserTest extends BaseLoggedTest {
+    @Test
+    @DisplayName("BasicLoginTest")
+    void loginTest() throws InterruptedException, ElementNotFoundException {
+        log.info("Starting test");
+        this.login(user, true);
+
+
+        WebElement logged_user = user.getDriver().findElement(By.xpath("//*[@id=\"logoutForm\"]/section[1]/div"));
+        Assertions.assertEquals("alice", logged_user.getText());
+        WebElement product = user.getDriver().findElement(By.xpath("/html/body/div/div[3]/div[3]/form/input[1]"));
```

Comment:
Xpath locators lead to complex expressions, I assume that you have checked that you can't get a simpler location, right?

Just another tip: Because PageObjects are not used for the tests, over the time you may need shorten the element finding statements, maybe in the superclass

Reply:
> Xpath locators lead to complex expressions, I assume that you have checked that you can't get a simpler location, right?
> 
> Just another tip: Because PageObjects are not used for the tests, over the time you may need shorten the element finding statements, maybe in the superclass

Yes, these elements don't have an ID that allows us to identify them by it. To shorten the reference to the element called, in Fullteaching, we utilize a [Constants](https://github.com/giis-uniovi/retorch-st-fullteaching/blob/main/src/test/java/com/fullteaching/e2e/no_elastest/common/Constants.java) file. This file serves to consolidate and prevent the repetition of IDs, Xpaths, and other attributes across multiple sites.

Reply:
IMO using a constants file is the worst option because it loses functional cohesion.

Maybe I didn't explained well, I wanted to say that later (in the future) it could be needed to shorten the expressions that find statements (user.getDriver().findElement()). I didn't mentioned the strings. This is not urgent for now

Reply:
All clear! I believe that on Sunday, I accidentally requested a revision because the 'green' button for sending is also used for the review request

Reply:
I'm not sure what this is the reason, because I still receive notifications for every commit (3 commits today), despite I changed the PR to draft, that is the standard procedure to avoid the spam...